packages/transaction-manager/src/transactions-factory.ts

```diff
@@ -136,6 +139,62 @@ export default class TransactionsFactory {
 
     const encryptedDataSerialized: string = MultiFormat.serialize(encryptedData);
 
-    return { encryptedData: encryptedDataSerialized };
+    if (encryptionParams.length === 0) {
+      return { encryptedData: encryptedDataSerialized };
+    } else {
+      // TODO: Refactor, duplicate with createEncryptedTransactionInNewChannel
```

Comment:
Do you plan to fix the TODO before merging? If not use `FIXME` instead

Reply:
Changed to FIXME